packages/supabase/.env.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/plotwist-dev
```

Comment:
Since this `DATABASE_URL` contains the "plotwist-dev" which is the database selected, could you also add a new variable to the `docker-compose.yml` file to match this default database name? Otherwise this database url should not connect (I guess, I don't know if it creates automatically when trying to connect).

Reply:
the database is created automatically, but I can add it there too, no problem